2025/README.md

```diff
@@ -1,18 +1,396 @@
+Interop 2025 is an effort to increase the interoperability of key features of the web platform across browsers. These features, which are of high priority to web developers and end users, have been [proposed](https://github.com/orgs/web-platform-tests/projects/5) by the web development community, and have been selected by the Interop Project team (to learn more, see [Interop 2025 proposal selection process](https://github.com/web-platform-tests/interop/blob/main/2025/selection-process.md)).
 
-## Proposals
+The Interop 2025 effort is part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project — an automated test suite for web standards — and run by a team of representatives from companies that make substantial contributions to browser rendering engines (including Apple, Bocoup, Google, Igalia, Microsoft, and Mozilla).
 
-Thank you to everyone who submitted a [proposal for Interop 2025](https://github.com/orgs/web-platform-tests/projects/5)! Please see the timeline below for the ongoing planning of Interop 2025.
+Similar to previous years ([2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and [2024](https://wpt.fyi/interop-2024)), the tests we selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the [Interop 2025 dashboard](https://wpt.fyi/interop-2025) — displaying the percentage of passing tests in each chosen area, and an overall total score of tests that pass in every browser. The dashboard will also display scores for group progress on particular investigation efforts selected by the Interop team to work on throughout the year.
 
-## Timeline
+Interop 2025 is focused on technology that is already specified in web standards.
 
-Interop 2025 will open for proposals on September 17, 2024 and the overall timeline is as below. Please note that the timelines are subject to change.
-- Proposal submission window (~3 weeks): Sept 17th, 2024 through Oct 9th, 2024
-- Proposal selection: Before December 19th 2024
-- Scope of the Interop 2025 project published: the first half of February, 2025.
+## Contents
 
-## Guidance for proposal authors
+* [Focus areas](#focus-areas)
+  * [Anchor positioning](#anchor-positioning)
+  * [`backdrop-filter`](#backdrop-filter)
+  * [Core Web Vitals](#core-web-vitals)
+  * [`<details>` element](#details-element)
+  * [Layout](#layout)
+  * [Modules](#modules)
+  * [Navigation API](#navigation-api)
+  * [Pointer and Mouse events](#pointer-and-mouse-events)
+  * [Remove mutation events](#remove-mutation-events)
+  * [`@scope`](#scope)
+  * [`scrollend` event](#scrollend)
+  * [Storage Access API](#storage-access-api)
+  * [`text-decoration`](#text-decoration)
+  * [URLPattern](#urlpattern)
+  * [View Transitions](#view-transitions)
+  * [WebAssembly](#webassembly)
```

Comment:
```suggestion
  * [WebAssembly Extensions](#webassembly)
```

It's only about two specific features AFAIK, not the whole of WASM:

* JS string builtins: to make the WebAssembly builtin string functions mirror a subset of the JavaScript String API so it can be callable without JavaScript glue code.
* Resizable buffers integration: to integrate WebAssembly into JS code that uses resizable buffers.

WabAssembly JS integration might work too? I'm open to different wording.